src/net.cpp

```diff
@@ -1625,7 +1626,7 @@ void CConnman::ThreadOpenConnections()
         {
             LOCK(cs_vNodes);
             BOOST_FOREACH(CNode* pnode, vNodes) {
-                if (!pnode->fInbound) {
+                if (!pnode->fInbound && !pnode->fAddnode) {
```

Comment:
This makes addnoded connections no longer prevent random connections to the same netgroups. Is that change implied by "which is not subject to any of the other connection limitations"?